api/src/test/groovy/ncei/onestop/api/service/MetadataParserTest.groovy

```diff
@@ -214,18 +213,18 @@ class MetadataParserTest extends Specification {
     def document = ClassLoader.systemClassLoader.getResourceAsStream("test-iso-metadata.xml").text
 
     when:
-    def spatialBounding = MetadataParser.parseSpatialInfo(document)
+    def result = MetadataParser.parseSpatialInfo(document)
 
     then:
-    spatialBounding == [spatialBounding     :
-            [
-                    type       : 'envelope',
-                    coordinates: [
-                            [-180, 90],
-                            [180, -90]
-                    ]
-            ],
-            isGlobal: true]
+    result == [
+        spatialBounding: [
+            type: 'polygon',
```

Comment:
Since we added support for points as well as polygons, should we add a test for that? Also, is there a reason we don't also have a test in here for a non-global example?

Reply:
Good idea. Added a test with a point geometry (which is also non-global).